.lintstagedrc.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
-  '*.(js|mjs|jsx|css|html|d.ts|js.flow)': 'prettier --write',
+  '*.(js|mjs|jsx|css|html|d.ts|js.flow|ts|tsx)': 'prettier --write',
   '*.(js|mjs|jsx)': ['flow focus-check', 'eslint --fix'],
```

Comment:
Do we even need these anymore? // @thegreatercurve

Reply:
You can remove the Flow check completely. We still use Eslint for the TS files so you can add that for the TS files:
```suggestion
  '*.(js|mjs|jsx|ts|tsx)': ['eslint --fix'],
```